api/V1/AuthenticateUser.js

```diff
@@ -22,6 +22,8 @@ const auth = require("../auth");
 const responseUtil = require("./responseUtil");
 const middleware = require("../middleware");
 
+const ttyTypes = Object.freeze({ short: 60, medium: 5 * 60, long: 30 * 60 });
```

Comment:
I think you mean ttl (time to live) not tty (which typically means teletype or terminal type)